dsl/src/main/scala/asobu/dsl/Syntax.scala

```diff
@@ -60,8 +63,6 @@ trait CompositionSyntax
     def respondJson(r: Results#Status)(implicit writes: Writes[RMT]): Directive[RMT] = respond(t ⇒ r.apply(Json.toJson(t)))
   }
 
-  def from[Repr <: HList] = Extractor.apply[Repr] _
-
   def fromAuthorized[AuthInfoT](ba: RequestHeader ⇒ Future[Either[String, AuthInfoT]]) = new AuthInfoExtractorBuilder[AuthInfoT](ba)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Type parameter 'AuthInfoT' does not conform with naming standard.](https://www.codacy.com/app/iHeartRadio/asobu/file/2260975636/issues/source?bid=3037259&fileBranchId=3178877#l66)